src/message_bus.zig

```diff
@@ -1,15 +1,19 @@
 const std = @import("std");
 const assert = std.debug.assert;
-const Allocator = std.mem.Allocator;
+const mem = std.mem;
 
 const vr = @import("vr.zig");
```

Comment:
According to the official zig style guide, namespaces such as vr should be snake_case. if vr were a struct with fields it would be TitleCase. Not that I really mind if you'd rather have `VR` than `vr`, but I think this convention is useful as namespaces can't be instantiated.

Reply:
Ah, thanks, I didn't realize that of course this was a namespace. Perfect as is.